recipe/meta.yaml

```diff
@@ -63,3 +63,4 @@ extra:
     - joshmoore
     - jrbourbeau
     - jhamman
+    - rabernat
```

Comment:
@rabernat have added you to the maintainers here. Please let me know whether that is ok

Reply:
:+1: Yes great, thanks!